### PR TITLE
fix: create course page fully connected to database

### PR DESCRIPTION
**Pull request by saidabdul80**
course can be created to database throught create course page. four tables on the database are feed with data from the create course page. 1. course table, 2. course experiment, 3. Course Resources, 4. Course Instructor tables

### CODE REVIEW

File: app/Http/Controllers/Api/CourseController.php [hunk 1]
```diff
@@ -9,52 +9,114 @@
...
+            $resource_uuid = Util::uuid();
+            $resourceUrl = $request->get('resource_url');
+
+            /*upload image*/
+            list($type, $img_data)  = explode(';', $resourceUrl);
+            list(, $img_data)  = explode(',', $img_data);
+            $decode_img = base64_decode($img_data);
+            $path = public_path().'/images/resources/';
+            if (!is_dir($path)){
+                File::makeDirectory($path,$mode =0777, true, true);
```
Comment (bynalab) [inline app/Http/Controllers/Api/CourseController.php:70]: We are not saving file on the system itself, rather using external source, right? @Auwalms

**Review by bynalab — COMMENTED** [review]
Please let me handle the code modification on the backend next time. Thanks

File: app/Http/Controllers/Api/CourseController.php (hunk 1 above)
Comment (Auwalms) [reply app/Http/Controllers/Api/CourseController.php:70]: Correct @bynalab. @saidabdul80 please revert this part of the PR. At the frontend level, you are expected to upload the resource (doc, pdf) to Amazon S3 (S3 Glacier gives us 10GB of free storage per month). Trying to not clutter our server or database with too many resources so the system doesn't get slow.

Comment (Auwalms) [reply app/Http/Controllers/Api/CourseController.php:70]: @saidabdul80 please update the PR as discussed above.  just the changes to the resource URL BTW

Comment (abdullahikawu) [reply app/Http/Controllers/Api/CourseController.php:70]: Ibid

**Review by Auwalms — APPROVED** [review]
👏👏👏👏

**Pull request merged**